app/assets/stylesheets/components/_user_info_item.sass

```diff
@@ -0,0 +1,10 @@
+$UserInfoItem-color-delete: Theme-color(button, delete)
+$UserInfoItem-spacing-delete: $Theme-spacing-small
+
+.UserInfoItem-info
```

Comment:
As discussed personally, I believe the reasons this component exist are not the best